installer-static.tpl.sh

```diff
@@ -0,0 +1,106 @@
+
+  gpg_key_url="https://download.tarantool.org/tarantool/{{ rtype }}/series-{{ tarantool_version }}/gpgkey"
+  gpg_key_url_modules="https://download.tarantool.org/tarantool/release/modules/gpgkey"
+  apt_source_path="/etc/apt/sources.list.d/tarantool_{{ tarantool_version }}_static.list"
+
+  curl -L "${gpg_key_url}" | apt-key add -
+  curl -L "${gpg_key_url_modules}" | apt-key add -
+
+  echo "deb https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/{{ rtype }}/series-{{ tarantool_version }}/linux-deb/ static main" > ${apt_source_path}
+  echo "deb https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/release/modules/linux-deb static main" >> ${apt_source_path}
+
+  mkdir -p /etc/apt/preferences.d/
+  echo -e "Package: tarantool\nPin: origin download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}\nPin-Priority: 1001" > /etc/apt/preferences.d/tarantool
+  echo -e "\nPackage: tarantool-common\nPin: origin download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}\nPin-Priority: 1001" >> /etc/apt/preferences.d/tarantool
+  echo -e "\nPackage: tarantool-dev\nPin: origin download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}\nPin-Priority: 1001" >> /etc/apt/preferences.d/tarantool
+
+  apt-get update
+
+  echo "Tarantool {{ tarantool_version }} is ready to be installed by 'apt-get -y install tarantool'"
+
+  if [[ ${TARANTOOL_AUTO_INSTALL:-False} = "True" ]]; then
+    DEBIAN_FRONTEND=noninteractive apt-get -y install tarantool
+  fi
+}
+
+setup_yum_dnf_repo ()
+{
+  cat <<EOF > /etc/yum.repos.d/tarantool_{{ tarantool_version }}_static.repo
+[tarantool]
+name=Tarantool
+baseurl=https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/{{ rtype }}/series-{{ tarantool_version }}/linux-rpm/static/\$basearch/
+gpgkey=https://download.tarantool.org/tarantool/{{ rtype }}/series-{{ tarantool_version }}/gpgkey
+repo_gpgcheck=1
+gpgcheck=0
+enabled=1
+priority=1
+
+[tarantool_modules]
+name=Tarantool modules
+baseurl=https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/release/modules/linux-rpm/static/\$basearch/
+gpgkey=https://download.tarantool.org/tarantool/release/modules/gpgkey
+repo_gpgcheck=1
+gpgcheck=0
+enabled=1
+priority=1
+EOF
+
+  if yum --version > /dev/null 2>&1; then
+    if [[ ${TARANTOOL_AUTO_INSTALL:-False} = "True" ]]; then
+      yum -y install tarantool
+    else
+      echo "Tarantool {{ tarantool_version }} is ready to be installed by 'yum install -y tarantool'"
+    fi
+  else
+    if [[ ${TARANTOOL_AUTO_INSTALL:-False} = "True" ]]; then
+      dnf -y install tarantool
+    else
+      echo "Tarantool {{ tarantool_version }} is ready to be installed by 'dnf install -y tarantool'"
+    fi
+  fi
+}
+
+setup_zypper_repo ()
+{
+  zypper ar -f https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/{{ rtype }}/series-{{ tarantool_version }}/linux-rpm/static/$(uname -m)/ tarantool_static
```

Comment:
I like adding the repository in such a way, but to be consistent with `setup_yum_dnf_repo` I suggest using this:

```bash
  rpm --import https://download.tarantool.org/tarantool/{{ rtype }}/series-{{ tarantool_version }}/gpgkey
  rpm --import https://download.tarantool.org/tarantool/release/modules/gpgkey
  cat <<EOF > /etc/zypp/repos.d/tarantool_{{ tarantool_version }}_static.repo
[tarantool]
name=Tarantool
baseurl=https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/{{ rtype }}/series-{{ tarantool_version }}/linux-rpm/static/\$basearch/
gpgkey=https://download.tarantool.org/tarantool/{{ rtype }}/series-{{ tarantool_version }}/gpgkey
repo_gpgcheck=1
gpgcheck=0
enabled=1
priority=1

[tarantool_modules]
name=Tarantool modules
baseurl=https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/release/modules/linux-rpm/static/\$basearch/
gpgkey=https://download.tarantool.org/tarantool/release/modules/gpgkey
repo_gpgcheck=1
gpgcheck=0
enabled=1
priority=1
EOF
```

Reply:
done